src/app/shared/constants/constants.ts

```diff
@@ -25,7 +26,7 @@ export class Constants {
   static readonly PHONE_PREFIX = '+380';
   static readonly PROVIDER_ENTITY_TYPE = 1;
   static readonly WORKSHOP_ENTITY_TYPE = 2;
-
+  static readonly ITEMS_PER_PAGE_DIR =10;
   static readonly ITEMS_PER_PAGE = 2 * Math.floor(window.innerWidth / (332));
```

Comment:
what do you think is it make sense to use the same approach as with ITEMS_PER_PAGE constant? I mean "2 * Math.floor(window.innerWidth / (332))"